draft-ietf-oauth-v2-1.md

```diff
@@ -208,7 +208,7 @@ protected resources hosted by the resource server.
 For example, an end-user (resource owner) can grant a printing
 service (client) access to their protected photos stored at a photo-
 sharing service (resource server), without sharing their username and
-password with the printing service.  Instead, they authenticates
+password with the printing service.  Instead, he authenticates
```

Comment:
I would prefer to stick with the singular "they" so I'm re-creating the rest of these typos in a new commit instead